src/lib/caching/structures/channels/NewsChannel.ts

```diff
@@ -16,6 +16,20 @@ export class NewsChannel extends GuildTextChannel {
 	 */
 	public readonly type = ChannelType.GuildAnnouncement;
 
+	/**
+	 * Follows this channel.
+	 * @param webhookChannel
+	 * @since 0.0.1
+	 */
+	public async follow(webhookChannel: GuildTextChannel): Promise<APIChannelFollowResult> {
```

Comment:
the parameter should probably be named channel instead of webhookChannel, as that gives the wrong idea.